sklearn/linear_model/ridge.py

```diff
@@ -921,6 +919,40 @@ def classes_(self):
         return self._label_binarizer.classes_
 
 
+def _check_gcv_mode(X, gcv_mode):
+    possible_gcv_modes = {None, 'auto', 'svd', 'eigen'}
+    if gcv_mode not in possible_gcv_modes:
+        raise ValueError(
```

Comment:
Apparently, we don't have a test for this case

Reply:
thanks!